e2e/test/scenarios/native/reproductions/15946-mongo-pre-select-table.cy.spec.js

```diff
@@ -17,10 +17,13 @@ describe.skip("issue 15946", { tags: "@external" }, () => {
   it("converting a question to the native query should pre-select a table (metabase#15946)", () => {
     cy.get(".QueryBuilder .Icon-sql").click();
 
-    modal().findByText("Convert this question to a native query").click();
-    modal().should("not.exist");
+    cy.get(".Modal")
```

Comment:
`modal()` expects `ModalContent` to be present in DOM, but `NativeQueryModal` uses `ModalBody` directly.

Please let me know if I should update `NativeQueryModal` instead (although [it looks like it does not use `ModalContent` on purpose](https://github.com/metabase/metabase/pull/27072)).

Reply:
Yes, on purpose